.changeset/cyan-fishes-punch.md

```diff
@@ -0,0 +1,5 @@
+---
+"@spectrum-css/toast": major
```

Comment:
@jawinn @castastrophe do we think this is a `major`? Feels like is maybe just a fix (`patch`)?